lib/core/test/unit/Cardano/Wallet/DBSpec.hs

```diff
@@ -1,14 +1,685 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
 module Cardano.Wallet.DBSpec
     ( spec
+    , DummyTarget
+    , DummyStateMVar (..)
```

Comment:
I had problems with Arbitrary instances needed in DbSpec after relocation. To avoid them I put every instances in DBSpec...Maybe it was too quick decision

Reply:
That's quite minor, but yes, I think at least `DummyStateMVar` could just be with the MVar specs.

Reply:
fair argument - done